TypeScript.YAML-tmLanguage

```diff
@@ -2671,7 +2671,11 @@ repository:
       endCaptures:
         '0': { name: punctuation.definition.comment.ts }
       patterns:
-      - include: '#docblock'
+      - contentName: comment.block.documentation.body.ts
+        begin: (?<=/\*\*)([^*]|\*(?!/))*$
+        while: (^|\G)\s*\*(?!/)(?=([^*]|[*](?!/))*$)
```

Comment:
The while pattern does not work in sublime and results in error and hence cannot be applied in this repo as it sublime typescript package also uses this grammar.

Reply:
Unfortunately is no way to implement this without begin/while. I can keep this grammar modification VS Code but it means that other consumers of this grammar (like VS) will not benefit